data.world.cmake

```diff
@@ -8,7 +8,7 @@ set(TARGET_NAME "${IRODS_TARGET_NAME_PREFIX}-${POLICY_NAME}")
 
 include(IrodsExternals)
 
-IRODS_MACRO_CHECK_DEPENDENCY_SET_FULLPATH_ADD_TO_IRODS_PACKAGE_DEPENDENCIES_LIST(ELASTICCLIENT elasticlient0.1.0-1)
+IRODS_MACRO_CHECK_DEPENDENCY_SET_FULLPATH_ADD_TO_IRODS_PACKAGE_DEPENDENCIES_LIST(ELASTICCLIENT elasticlientd68e30e3-0)
```

Comment:
This is because the externals package references unreleased code?

Reply:
Looks released to me https://packages.irods.org/apt/pool/focal/main/i/irods-externals-elasticlientd68e30e3-0/

Reply:
Oh. This repo is just behind.

Reply:
More context for the change: d68e30e3 is the current version in the externals repo, and is the only version in the repo for Ubuntu 20.

Reply:
Got it. Please mark this as resolved.